graal/docker/oap/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG BASE_IMAGE='eclipse-temurin:11-jre'
+
+ARG SKYWALKING_CLI_VERSION
+
+FROM apache/skywalking-cli:$SKYWALKING_CLI_VERSION as cli
+
+FROM  --platform=linux/amd64 ubuntu:latest
```

Comment:
So we only support amd64 architecture in our Docker images? We don't have to add `arm` support in this PR, just wondering whether there is any blocker to support arm64 arch

Reply:
ARM architecture should can be supported, but for the sake of passing tests stably, I specified AMD (some Docker images do not support ARM architecture operating systems).

Reply:
Let's open another PR to verify this.
Considering timing of OSPP, I could skip this.